docs/administrator-handbook.md

```diff
@@ -137,6 +132,51 @@ changes from the pending deployment to the booted deployment.
 
 See `man rpm-ostree` for more information.
 
+## Using overrides and `usroverlay`
+
+While some people talk about "immutability" when referring to image-based
+systems like rpm-ostree, in fact a top level goal of rpm-ostree is
+to *empower* users and system administrators.  When something goes
+wrong, you are root on your own computer and should have the ability to apply
+overrides locally.
+
+First, there is the `rpm-ostree override replace` command, which
+will replace an RPM, and apply that change persistently for the *next*
+boot - this is symmetric with how `rpm-ostree install` works.
+
+For example, suppose you want to test a fix to `podman`.  You can pass
+both direct HTTP URLs as well as local files:
+
+```
+$ rpm-ostree override replace https://kojipkgs.fedoraproject.org//packages/podman/3.3.1/1.fc34/x86_64/podman-3.3.1-1.fc34.x86_64.rpm
+```
+
+Now, suppose that you want to test this change *live*.  There are two choices.
+The first choice is to run
+
+```
+$ rpm-ostree ex apply-live --allow-replacement
+```
+
+This is a currently experimental interface that will pull the pending
+changes and apply them live.  You can `rpm-ostree ex apply-live --reset`
+to revert back to the booted tree.
+
+The second choice is `rpm-ostree usroverlay` which creates a transient writable `overlayfs` over `/usr` where you
+can do anything, such as for example use `rpm -Uvh https://kojipkgs.fedoraproject.org//packages/podman/3.3.1/1.fc34/x86_64/podman-3.3.1-1.fc34.x86_64.rpm`.
+
+The changes here will not persist across reboots, which makes this a great choice for
+testing.
+
+One downside though is there's no equivalent of `rpm-ostree ex apply-live --reset`
+today when `rpm-ostree usroverlay` is in place.  It's possible to find the original
+binaries in a previous deployment, or via `ostree checkout` of the base commit, etc.
```

Comment:
Let's mention `override reset` somewhere in this section to be similar to the "Removing a base package" section below?

Reply:
I don't quite follow how that's related?

Reply:
Sorry, should've been clearer. In this section, we talk about `override replace` but don't mention that to stop overriding a replacement that you have to use `override reset`. We do do this for `override remove` in the section below though, but to be explicit it'd be nice to also mention it here so it's clear that `override reset` is to reset both "replace" and "remove" type overrides.

Reply:
OK, I tried redoing this again.